apps/backend/src/main/resources/api.yaml

```diff
@@ -1,22 +1,985 @@
 openapi: "3.0.0"
 
 info:
-  title: "storeme cloud API"
+  title: "StoreMe API"
   version: "1"
 
 servers:
   - url: http://localhost:8091
-    description: catalogue api
+    description: Main API
+
+tags:
+  - name: Item
+    description: Relates to items
+  - name: ItemModel
+    description: Relates to item models
+  - name: Location
+    description: Relates to locations
+  - name: Movement
+    description: Relates to movements
+  - name: User
+    description: Relates to users
+  - name: Inventory
+    description: Relates to inventories
+  - name: Jwt
+    description: Relates to JWT
 
 paths:
-  /example:
+  /authentication/getToken:
+    post:
+      tags:
+        - Jwt
+      summary: Get token
+      description: Get a JWT token for a username and password.
+      operationId: getToken
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/CredentialsDTO'
+      responses:
+        '200':
+          description: Operation successful.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/JWTDTO'
+        400:
+          $ref: '#/components/responses/400'
+        401:
+          $ref: '#/components/responses/401'
+        403:
+          $ref: '#/components/responses/403'
+        404:
+          $ref: '#/components/responses/404'
+        409:
+          $ref: '#/components/responses/409'
+        500:
+          $ref: '#/components/responses/500'
+  /authentication/validateToken:
+    post:
+      tags:
+        - Jwt
+      summary: Validate token
+      description: Verify that a token can be used.
+      operationId: validateToken
+      parameters:
+        - name: Authorization
+          in: header
+          schema:
+            type: string
+      responses:
+        '200':
+          description: If the JWT is valid.
```

Comment:
204 if no content returned.

It is a 401 if the token is no longer valid?

Reply:
C.F description of the response 
> No authorization was provided or token is invalid.

Either no longer valid, forged or simply invalid in terms of content